src/Core/Geographic/Extent.js

```diff
@@ -72,27 +72,44 @@ Extent.prototype.isTiledCrs = function fnIsTiledCrs() {
     return this._crs.indexOf('WMTS:') == 0 || this._crs == 'TMS';
 };
 
+const s = 20037508.34 * 2;
```

Comment:
What is this ? Please add a comment so we don't forget

Reply:
added